pdsfile/pds4file/rules/cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,280 @@
+##########################################################################################
+# Update the global dictionary of subclasses
+##########################################################################################
+
+pds4file.Pds4File.SUBCLASSES['cassini_uvis_solarocc_beckerjarmak2023'] = cassini_uvis_solarocc_beckerjarmak2023
+
+##########################################################################################
+# Unit tests
+##########################################################################################
+
+import pytest
+from .pytest_support import *
```

Flake8 (7.2.0)

[error] 'from .pytest_support import *' used; unable to detect undefined names

(F403)